RELEASE/scripts/autoscend/auto_pre_adv.ash

```diff
@@ -280,11 +282,19 @@ void handlePreAdventure(location place)
 			abort("Tried to charge a WineBomb but don't have one.");
 		}
 		autoEquip($slot[off-hand], $item[Unstable Fulminate]);
+		addToMaximize("+equip " + $item[Unstable Fulminate]);
 	}
 
-	if(place == $location[The Black Forest])
+	if((place == $location[The Black Forest]) && (my_turncount() != 0)
```

Comment:
why the turncount check? Dont see how we might end up trying to go to black forrest or desert on turn 0. If it did it would fail anyway because they are inaccessible. I know we change location in mafia without adventuring sometime (going to noob cave for god lobster fight for example), maybe you know a quirk I dont.